chanjo/__init__.py

```diff
@@ -8,6 +8,10 @@
 :copyright: (c) 2014 by Robin Andeer
 :licence: MIT, see LICENCE for more details
 """
+
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
This is my only question really

Why not make this a root logger (``logging.getLogger()``) which will then automatically format loggers in the external dependency packages?